01/nightwarrior-xxx/wordvalue.py

```diff
@@ -0,0 +1,32 @@
+
+from data import DICTIONARY, LETTER_SCORES
+
+def load_words():
+    llist=[]
+    fp=open('dictionary.txt','r')
+    for i in fp.readlines():
+        llist.append(i.strip())
+    fp.close()    
+    return llist
+    
+
+def calc_word_value(word):
+    sum=0
+    for j in word:
+        sum=sum+(LETTER_SCORES.get(j.upper(),0))
```

Comment:
good, you can give `sum` an iterator: `return sum(LETTER_SCORES.get(char.upper(), 0) for char in word)`